ged2dot.py

```diff
@@ -727,7 +727,7 @@ def calc(self) -> None:
 
 class DescendantsLayout(Layout):
     """A layout that shows all descendants of a root family."""
-    def filter_families(self) -> List[Family]:
+    def filter_families(self) -> None:
```

Comment:
This looks problematic: DescendantsLayout.filter_families() overwrites default behavior from Layout.filter_families(). So either update both or none. CI nicely caught this problem.